Week2/homework/js-exercises/4-the-reading-list.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+var books = [{
+        title: 'The Design of EveryDay Things',
+        author: 'Don Norman',
+        alreadyRead: false
+    },
+    {
+        title: 'The Most Human Human',
+        author: 'Brian Christian',
+        alreadyRead: true
+    },
+    {
+        title: 'The Lord of the Rings',
+        author: 'J. R. R. Tolkien',
+        alreadyRead: true
+    }
+];
+
+for (var i = 0; i < books.length; i++) {
+    var book = books[i];
+    var bookInfo = book.title + '" by ' + book.author;
+    if (book.alreadyRead) {
+        console.log('You already read "' + bookInfo);
+    } else {
+        console.log('You still need to read "' + bookInfo);
+    }
+}
```

Comment:
👍 but `let` instead of `var` please

Reply:
thank you so much Yash-kapila for the feedback an I will change the var to let.